store/aciinfo.go

```diff
@@ -21,46 +21,57 @@ import (
 	"time"
 )
 
+// ACIFetchInfo is used to store information about the fetching process of an ACI.
+type ACIFetchInfo struct {
```

Comment:
I'm not too happy about it, but we can easily refactor it later on. I implemented your suggestion.